code/game/turfs/simulated/wall/mineral_walls.dm

```diff
@@ -153,8 +153,14 @@
 	icon = 'icons/turf/walls/snow_wall.dmi'
 	icon_state = "snow"
 	hardness = 80
+	explosion_block = 0
+	slicing_duration = 30
 	sheet_type = /obj/item/stack/sheet/mineral/snow
 	canSmoothWith = null
+	girder_type = null
+	
+/turf/closed/wall/mineral/snow/devastate_wall()
```

Comment:
Hey could you make a `/turf/closed/wall/mineral/girderless` intermediate type to implement this behavior?  I've been meaning to get pure-sandstone walls in.